AppleALC.xcodeproj/project.pbxproj

```diff
@@ -1132,7 +1289,7 @@
 				CODE_SIGN_IDENTITY = "";
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = "$(MODULE_VERSION)";
-				DEPLOYMENT_POSTPROCESSING = YES;
+				DEPLOYMENT_POSTPROCESSING = NO;
```

Comment:
Do these disable stripping for all archs? Might be not nice for file size.

Reply:
Added equivalent strip command in https://github.com/acidanthera/AppleALC/commit/41aff415890a44c76909cc487bc37f4c555e4617. Deployment postprocessing must be disabled to ensure the binary is not modified after scripts are run.